### PR TITLE
minor improvements to some text and formatting

### PR DESCRIPTION
**Pull request by nlharris**
QUESTION: Is there a remote participation option? If so, can you mention in this document how to find out how to join? (Don't put the Zoom URL on this public page, of course.)

Resolves [#ISSUE, #ISSUE]

- [ ] `docs/` have been added/updated if necessary
- [ ] `make test` has been run locally
- [ ] tests have been added/updated (if applicable)
- [ ] [CHANGELOG.md](https://github.com/mapping-commons/sssom/blob/master/CHANGELOG.md) has been updated.

If you are proposing a change to the SSSOM metadata model, you must 

- [ ] provide a full, working and valid example in `examples/`
- [ ] provide a link to the related GitHub issue in the `see_also` field of the linkml model
- [ ] provide a link to a valid example in the `see_also` field of the linkml model

[Add a description, mentioning at least relevant #ISSUE and how it was addressed. A bulleted list of all changes performed by the PR is is helpful.]

### CODE REVIEW

File: src/docs/events/mc2023.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
-# 2nd Mapping Commons Workshop on Simple Standard for Sharing Ontology Mappings
+# 2nd Mapping Commons Workshop on Simple Standard for Sharing Ontology Mappings (SSSOM)
 
-When: Sunday, 23rd April, Time: 2:30 pm Italy (CET), 1:30 pm BST, 8:30 am EDT, 5:30 am PDT (3 hours total)
-Where: co-located with [Biocuration 2023](https://biocuration2023.github.io/). For information about the venue see https://biocuration2023.github.io/workshops.
```
Comment (nicolevasilevsky) [inline src/docs/events/mc2023.md:4]: i don't think we want to remove the locayion

**Comment by nlharris** [comment]
Thanks!

 Is there a remote participation option? If so, can you mention in this document how to find out how to join? (Don't put the Zoom URL on this public page, of course.)

**Comment by nicolevasilevsky** [comment]
there was but this workshop already happened.

**Comment by nlharris** [comment]
Ohhh! 😊

**Pull request merged**